docs-gastby/src/pages/demo/index.mdx

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
This file (and `demo` directory) can be removed, since we don't want to publish template pages that are not relevant to the UI.